service/controller/resource/promxy/servergroup.go

```diff
@@ -0,0 +1,172 @@
+package promxy
+
+import (
+	"fmt"
+	"time"
+
+	config_util "github.com/prometheus/common/config"
+	"github.com/prometheus/common/model"
+	"github.com/prometheus/prometheus/discovery/kubernetes"
+	"github.com/prometheus/prometheus/pkg/relabel"
+)
+
+// Config is the configuration for a ServerGroup that promxy will talk to.
+// This is where the vast majority of options exist.
+type ServerGroup struct {
```

Comment:
Question, would it not be better to import this from [github.com/jacksontj/promxy/pkg/servergroup](https://github.com/jacksontj/promxy/blob/master/pkg/servergroup/config.go)?